### DISCUSSION TITLE
How to update Starship.

### DISCUSSION THREAD

**Discussion by radam9** [starship/starship#2239] [Q&A] [answered]
Hey there,

I have been looking all over the documentation, issues and google, but I can't find how I am supposed to update starship.

Any guidance would be greatly appreciated.
Thanks

**Comment by matchai** [comment, marked as answer]
Hey @radam9 👋 

We suggest folks use the same process as they did to install Starship when updating it. That is, if you used a package manager, that should be able to keep Starship up-to-date. Otherwise, if you used the `curl | bash` command we provide, rerunning the command should update Starship in-place.

**Reply by radam9** [reply]
great, thanks a lot.
PS: a small section or comment in the documentation would make it a lot easier for people like me :)
